contracts/KetlAttestation.sol

```diff
@@ -197,9 +207,8 @@ contract KetlAttestation is
     );
     // Check if this attestation has already been used
     require(
-      attestationHashesEntangled[attestationHash].current() == 0 ||
-        attestationHashesEntangled[attestationHash].current() <
-        maxEntanglementsPerAttestation[attestationHash],
+      attestationHashesEntangled[attestationHash].current() <
```

Comment:
But what if it doesn't exist? Maybe we shouldn't remove `|| 0`?

Reply:
You can do:
```sol
attestationHashesEntangled[attestationHash].current() || 0 < maxEntanglementsPerAttestationType[attestationType],
```
I think

Reply:
Solidity doesn't have the same problem as js where it would be `undefined` since `0` is the default value.